bip-0158.mediawiki

```diff
@@ -279,7 +279,8 @@ following items for each transaction in a block:
 * The scriptPubKey of each output, aside from all <code>OP_RETURN</code> output
   scripts.
 
-Any "nil" items MUST NOT be included into the final set of filter elements.
+Any "nil" items MUST NOT be included into the final set of filter elements. A
```

Comment:
I'm not sure this is necessary, as it is implied by the construction. However, if you think it's useful to clarify, I'd include the sentence at the end of the "Construction" section, not here.

Reply:
Moved.